test/index.js

```diff
@@ -1124,6 +1124,29 @@ describe('expect()', () => {
                 Hoek.assert(!exception, exception);
                 done();
             });
+
+            it('asserts called with only one argument', (done) => {
+
+                let exception0 = false;
```

Comment:
Ordinarily I would have written something like

```js
for ([const test of {name: "no", test: [], {name: "two", test: ['a', 'b']}, {name: "three", test: ['a', 'b', 'c']}]) {

  it('assertion fails when called with ${test.name} arguments', (done) => {

   let exception = false;
   try {
    Code.expect('abc').to.include.apply(to, test.arguments);
   }
   catch (err) {
    exception = err;
   }

   Hoek.assert(exception0.message === 'Can only assert include with a single parameter', exception);
   done();
  }
}
```

but I wasn't quite sure if this was to your style or not. Up to you. Let me know either way and I'll change it.